src/NeuralNetwork/index.js

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2019 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/*
+Generic NeuralNetwork class
+*/
+
+import * as tf from '@tensorflow/tfjs';
+
+class NeuralNetwork {
+  /**
+   * Create a Neural Network.
+   * @param {object} options - An object with options.
+   */
+  constructor(options) {
+    // TODO: create the model based on many more options and defaults
+
+    const inputUnits = options.input || 2;
+    const hiddenUnits = Math.floor(inputUnits / 2) + 1;
+    const outputUnits = options.output || 1;
+
+    this.model = tf.sequential();
+    const hidden = tf.layers.dense({
+      units: hiddenUnits,
+      inputShape: [inputUnits],
+      activation: 'sigmoid',
```

Comment:
TODO: add activation function to `options`

Reply:
added